package.json

```diff
@@ -25,20 +25,22 @@
     "flow:dev": "flow stop; flow start; chokidar .flowconfig flow/ src/ tests/ -i flow/logs/flow.log -c 'flow status' --initial",
     "stylelint": "stylelint --syntax scss **/*.scss",
     "lint": "npm run eslint && npm run stylelint",
-    "start": "npm run version-check && NODE_PATH='./:./src' node bin/server.js",
-    "start-func-test-server": "node bin/start-func-test-server.js",
+    "node": "NODE_PATH='./:./src' NODE_ICU_DATA='./node_modules/full-icu' node",
+    "start": "npm run version-check && npm run node bin/server.js",
+    "start-func-test-server": "npm run node bin/start-func-test-server.js",
     "test-ci": "bin/config-check.js && better-npm-run test-ci && cat ./coverage/lcov.info | coveralls",
     "test": "bin/config-check.js && jest --watch",
     "test-coverage": "bin/config-check.js && jest --coverage --watch",
     "test-coverage-once": "bin/config-check.js && jest --coverage",
     "test-once": "bin/config-check.js && jest && npm run lint",
-    "version-check": "node bin/version-check.js",
+    "version-check": "npm run node bin/version-check.js",
```

Comment:
Since this just checks the version it probably doesn't really need to run with icu support.